src/yardstick/__init__.py

```diff
@@ -41,6 +41,14 @@ def compare_results_against_labels(  # pylint: disable=too-many-arguments
 ]:
     descriptions = list(descriptions)
 
+    # this is a description of what was done to the results before comparison
```

Comment:
I'm not sure I completely understand this comment, so I wanted to try to summarize:

When we're comparing two result sets, if they have different fundamental configurations (like the latest year CVEs are allowed to be from), they're not really comparable. So each result set should remember these fundamental configs, and report them when being compared to other result sets. Is that right?

Reply:
that's right 👍